rust/ql/test/library-tests/dataflow/global/main.rs

```diff
@@ -91,6 +91,37 @@ fn data_through_method() {
     sink(b); // $ hasValueFlow=4
 }
 
+use std::ops::Add;
+
+struct MyInt {
+    value: i64,
+}
+
+impl Add for MyInt {
+    type Output = MyInt;
+
+    fn add(self, other: MyInt) -> MyInt {
+        MyInt { value: self.value }
+    }
+}
+
+pub fn test_operator_overloading() {
+    let a = MyInt { value: source(5) };
+    let b = MyInt { value: 2 };
+    let c = a + b;
+    sink(c.value); // $ MISSING: hasValueFlow=5
+
+    let a = MyInt { value: source(6) };
+    let b = MyInt { value: 2 };
+    let d = b + a;
```

Comment:
Wouldn't this make the difference compared to the above easier to spot when quickly scanning the test:
```suggestion
    let a = MyInt { value: 2 };
    let b = MyInt { value: source(6) };
    let d = a + b;
```